packages/wonder-blocks-birthday-picker/src/components/__tests__/birthday-picker.test.js

```diff
@@ -340,4 +342,90 @@ describe("BirthdayPicker", () => {
             expect(onChange).toHaveBeenCalledTimes(1);
         });
     });
+
+    describe("labels", () => {
+        const translatedLabels: $Shape<Labels> = {
```

Comment:
`$Partial<Labels>`?

Reply:
@somewhatabstract Flow is throwing an error. How should I define the `Labels` prop if I use `$Partial` here?

Reply:
Hmm, what error is it throwing?

Reply:
This is one of the errors:
```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈
packages/wonder-blocks-birthday-picker/src/components/__tests__/birthday-picker.test.js:372:29

Cannot create BirthdayPicker element because undefined [1] is incompatible with string [1] in
property labels.day. [incompatible-type]

     packages/wonder-blocks-birthday-picker/src/components/__tests__/birthday-picker.test.js
     369│             render(
     370│                 <BirthdayPicker
     371│                     onChange={() => {}}
     372│                     labels={translatedLabels}
     373│                 />,
     374│             );
     375│

     packages/wonder-blocks-birthday-picker/src/components/birthday-picker.js
 [1]  31│     +day: string,
```

Reply:
Ah, I see. Yeah, that makes sense, because partial doesn't guarantee the existence of any specific field, they are all falsy. Whereas you want the ones you provided to be non-maybe typed...I guess $Shape is the right thing then in this situation.